Assets/MRTK/Examples/Demos/Gltf/Scripts/TestGltfLoading.cs

```diff
@@ -74,6 +74,6 @@ private async void Start()
                 Debug.Log("Import successful");
             }
         }
-        
+
```

Comment:
nit: this line could be removed

Reply:
I can take a look to see if there are some other rules we can add for these things (i.e. removing extra lines when collapsing closed braces